### PR TITLE
Refactor hats

### PR DESCRIPTION
**Pull request by horiaradu**

### CODE REVIEW

File: src/components/Hat.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styles from "./Hat.module.css";
+
+const Hat = ({ hat, imageStyle }) => {
+  return (
+    <div className={styles.image}>
+      <img
+        className={imageStyle || "responsive-image"}
+        alt={hat}
+        src={require("../static/images/hat.png").default}
```
Comment (PaulMacinic) [inline src/components/Hat.js:10]: In here you meant `${hat}.png`?

File: src/components/HatSlider.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    arrows: true,
+    speed: 250,
+    adaptiveHeight: true,
+    adaptiveWidth: true,
+    afterChange: (hat) => onHatSelected(hat),
+  };
+
+  console.log(selectedHat);
```
Comment (PaulMacinic) [inline src/components/HatSlider.js:19]: Let's remove the `clog`

**Review by PaulMacinic — CHANGES_REQUESTED** [review]
See comments

File: src/components/Hat.js (hunk 1 above)
Comment (horiaradu) [reply src/components/Hat.js:10]: not yet. only after u add 10 pictures :P

**Pull request merged**